modules/ROOT/pages/values-and-types/property-structural-constructed.adoc

```diff
@@ -50,23 +50,55 @@ In Neo4j, all relationships have a direction.
 However, you can have the notion of undirected relationships at query time.
 ====
 
-[[composite-types]]
-== Composite types
+[[constructed-types]]
+== Constructed types
 
-The following data types are included in the composite types category: `List` and `Map`.
+The following data types are included in the constructed types category: `LIST` and `MAP`.
 
-* Composite types can be returned from Cypher queries.
-* Composite types can be used as xref::syntax/parameters.adoc[parameters].
-* Composite types cannot be stored as properties (with the exception of homogenous lists).
-* Composite types can be constructed with xref::syntax/expressions.adoc[Cypher literals].
+* Constructed types can be returned from Cypher queries.
+* Constructed types can be used as xref::syntax/parameters.adoc[parameters].
+* Constructed types cannot be stored as properties (with the exception of homogenous lists).
+* Constructed types can be constructed with xref::syntax/expressions.adoc[Cypher literals].
 
-The `List` data type can be either a homogenous collection of simple values, or a heterogeneous, ordered collection of values, each of which can have any property, structural or composite type.
+The `LIST` data type can be either a homogenous collection of simple values, or a heterogeneous, ordered collection of values, each of which can have any property, structural or constructed type.
 
-The `Map` data type is a heterogeneous, unordered collection of (_Key_, _Value_) pairs, where _Key_ is a literal and  _Value_ can have any property, structural, or composite type. 
+The `MAP` data type is a heterogeneous, unordered collection of (_Key_, _Value_) pairs, where _Key_ is a literal and  _Value_ can have any property, structural, or constructed type.
 
-Composite values can also contain `null`. 
+Constructed type values can also contain `null`.
 For more details, see xref::values-and-types/working-with-null.adoc[working with null].
 
+[[types-synonyms]]
+== Types and their synonyms
+
+The table below shows the types and their syntactic synonyms.
+These types (and the synonyms) can be used in the type predicate expressions and the property type constraints.
+However, not all types can be used in all places.
+// TODO: add link to type predicate expression section (once exists)
+// TODO: add link to property type constraint section (once exists)
+Some of these types are not yet supported in either type predicate expressions nor property type constraints.
```

Comment:
I believe "nor" is a gramatical error here?
"A or B" -> `A || B`
"Either A or B" -> `A ^ B` (xor)
"Neither A nor B" -> `!A && !B`

I believe what you are trying to write is that they are all supported in at least one of them, but only a subset are supported in both? In that case I would write 
```suggestion
Some of these types are not yet supported in either type predicate expressions or property type constraints.
```

Reply:
No at the moment what we want to say is that a subset is at the moment not covered in any of them (there are for example nodes and maps which did not make the cut for the expressions either)

So it's probably more of
```suggestion
Some of these types are not yet supported in neither type predicate expressions nor property type constraints.
```

Reply:
That sounds wrong to me ^^

I don't think you should be using neither/nor when you have not already in the sentence. So Lasse's suggestion of either/or sounds better to me (just a feeling, don't know the grammar rules)

Reply:
let's see what the docs team thinks then before updating